payjoin/src/receive/v1/exclusive/mod.rs

```diff
@@ -23,13 +23,13 @@ pub fn build_v1_pj_uri<'a>(
 
 impl UncheckedProposal {
     pub fn from_request(
-        body: impl std::io::Read,
+        body: &[u8],
```

Comment:
This will be helpful! I was having a really tough time figuring out how to get a `Read` across the ffi layer in the v2_to_v2 python integration test